### ISSUE TITLE
Method not flagged as missing argument

### ISSUE THREAD

**Issue by Gabriel-p** [jsh9/pydoclint#301]
I'm running

    pydoclint --style=sphinx --check-style-mismatch=True . --exclude="(.venv|helpers*|docs)"

on my codebase and the following method which is missing the description for the `filename` argument

```
    def to_csv(self, filename: str) -> None:
        """Save data to a CSV file"""
        ...
```

is not flagged. Should this not raise a

    DOC101 Docstring contains fewer arguments than in function signature.
